kubernetes-schema.sql

```diff
@@ -1,11 +1,15 @@
 CREATE TABLE "Cluster" (
-  "cluster_id" int PRIMARY KEY,
+  "cluster_event_id" int PRIMARY KEY,
+  "cluster_id" int,
+  "timestamp" datetime NOT NULL;
```

Comment:
Data Types: In PostgreSQL, the equivalent of the datetime data type is timestamp. Please replace all instances of datetime with timestamp
Syntax Error: There are semicolons after the timestamp fields in the Cluster, Node, Pod, and Container tables, which should be replaced by commas

Reply:
Thanks for the change request. You're totally right. I corrected both points.